EjercicioClases/Controller/MainController.pm

```diff
@@ -49,99 +43,40 @@ sub startApp{
 }
 
 sub createSeller{
-    my @sellerData = View->inputForm();
-    if(isDuplicatedId($sellerData[0]) eq 'no' && isValidEmail($sellerData[2]) eq 'yes'){
-        my $seller = Model::User::Seller->new($sellerData[0], $sellerData[1], $sellerData[2], $sellerData[3], $sellerData[4]);
-        Controller::DatabaseController->saveSeller($seller);
-        #saveSeller($seller);
+    my @sellerData = View::View->inputForm();
+    my $seller = Model::User::Seller->new($sellerData[0], $sellerData[1], $sellerData[2], $sellerData[3], $sellerData[4]);
+    if($seller->isDuplicatedId($seller->getId) eq 'no' && $seller->isValidEmail($seller->getEmail)){
+        $seller->save($seller);
     }
```

Comment:
Here you can show a message, just to confirm the transaction was either successful or failed.